bsyncviewer/management/commands/bedes.py

```diff
@@ -49,7 +49,7 @@ def handle(self, *args, **options):
         if save_to_db:
             self.stdout.write('Reparsing BEDES data and Saving BEDES mappings to database')
```

Comment:
@haneslinger Would you modify the printout description and the notes below since we are not reparsing anymore with this command?
```suggestion
            self.stdout.write('Saving BEDES mappings to database')
            # if save_to_db is True, save CSV files to DB (no reparse)
```

Maybe we need to modify the readme page to better describe this whole workflow as well (especially the step of adding corresponding enumeration.json of BuildingSync schema).